### ISSUE TITLE
timer countdown depends on old flutter linter

### ISSUE THREAD

**Issue by plato79** [appinioGmbH/flutter_packages#173]
I cannot upgrade flutter_lints to 3.0.0 because timer_countdown depends on 2.0 version. 

```
Resolving dependencies...
Because <myapp> depends on flutter_timer_countdown ^1.0.5 which depends on flutter_lints ^2.0.0, flutter_lints ^2.0.0 is required.
So, because <myapp> depends on flutter_lints ^3.0.0, version solving failed.


You can try one of the following suggestions to make the pubspec resolve:
* Consider downgrading your constraint on flutter_timer_countdown: flutter pub add flutter_timer_countdown:^1.0.3
* Consider downgrading your constraint on flutter_lints: flutter pub add dev:flutter_lints:^2.0.3
exit code 1
```

**Comment by dhndeveloper** [comment]
same

**Comment by bilalhamud** [comment]
Solved with latest version 1.0.7
